docs/www/acls.md

```diff
@@ -0,0 +1,304 @@
+
+```cmd
+$ rpk topic create test-topic \
+  --tls-key <path to PEM-formatted key file> \
+  --tls-cert <path to PEM-formatted cert file> \
+  --tls-truststore <path to PEM-formatted truststore file>
+
+Created topic 'test-topic'.
+You may check its config with
+
+rpk topic describe 'test-topic'
+```
+
+> `rpk` defaults to connecting to `localhost:9092`. If you're connecting to a remote broker, you will need to pass `--brokers <node IP>:<kafka API port>`
+
+> For an in-depth guide about enabling TLS and mutual TLS in redpanda check out [our guide on the subject](https://vectorized.io/blog/tls-config/).
+
+### SASL/ SCRAM
+
+Redpanda also supports client authentication via SASL/ SCRAM (that is, the Simple Authentication and Security Layer protocol, using the Salted Challenge Response Authentication Mechanism), which is based on usernames & passwords.
+
+To enable it, set `redpanda.enable_sasl` to `true` in the configuration file, as well as list at least one "superuser" which after created will have permissions for all operations on the clusters.
+
+```yaml
+redpanda
+  
+  enable_sasl: true
+  superusers:
+  - admin
+  # The rest of the config...
+```
+
+Then, set a password for the user by running the following command. Replace `<password>` for a password of your choice. 
+
+```cmd
+$ rpk acl user create \
+  --new-username admin \
+  --new-password <password>
+
+Created user 'admin'
+```
+
+> Refer to the next section, _Managing users_, for more details on `rpk acl user`.
+
+You're now able to use the created identity to interact with the Kafka API. For example:
+
+**Note**: If you still have the TLS config from the previous section, you'll also need to pass the TLS flags.
+
+```cmd
+$ rpk topic describe test-topic \
+  --user admin \
+  --password <password> \
+  --brokers localhost:9092
+
+  Name                test-topic  
+  Internal            false       
+  Cleanup policy      delete      
+  Config:             
+  Name                Value       Read-only  Sensitive  
+  partition_count     1           false      false      
+  replication_factor  1           false      false      
+  cleanup.policy      delete      false      false      
+  Partitions          1 - 1 out of 1  
+  Partition           Leader          Replicas   In-Sync Replicas  High Watermark  
+  0                   1               [1]        [1]               0               
+```
+
+#### **Managing users**
+
+While having a superuser is a must to get started, it's not really a good idea to either share the superuser's credentials everywhere or to make everyone a superuser.
+
+You can create, delete and list users with `rpk acl user`.
+
+**Creating a user**
+
+```cmd
+$ rpk acl user create \
+  --new-username Jack \
+  --new-password <password>
+
+Created user 'Jack'
+```
+
+**Deleting a user**
+
+```cmd
+$ rpk acl user delete \
```

Comment:
Similarly with user creation, I had to use `--api-urls`. 

Note: the CLI would claim the user was deleted either way.